packages/lambdas/src/ihe-gateway-v2-scheduled-report.ts

```diff
@@ -0,0 +1,42 @@
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+import { getSecretValueOrFail } from "@metriport/core/external/aws/secret-manager";
+import { initReadonlyDbPool } from "@metriport/core/util/pg";
+import {
+  generatePatientDiscoveryReport,
+  generateDocumentQueryReport,
+  generateDocumentRetrievalReport,
+} from "@metriport/core/external/carequality/ihe-gateway-v2/report/report";
+import { capture } from "./shared/capture";
+import * as Sentry from "@sentry/serverless";
+
+// Keep this as early on the file as possible
+capture.init();
+
+const dbCredsArn = getEnvVarOrFail("DB_CREDS");
+const dbReadOnlyEndpoint = getEnvVarOrFail("DB_READ_REPLICA_ENDPOINT");
+const region = getEnvVarOrFail("AWS_REGION");
+
+capture.setExtra({ lambdaName: "scheduled-report-lambda" });
+
+export const handler = Sentry.AWSLambda.wrapHandler(async () => {
+  try {
+    const dbCreds = await getSecretValueOrFail(dbCredsArn, region);
+    const pg = initReadonlyDbPool(dbCreds, dbReadOnlyEndpoint);
+
+    const [patientDiscoveryReport, documentQueryReport, documentRetrievalReport] =
+      await Promise.all([
+        generatePatientDiscoveryReport(pg),
+        generateDocumentQueryReport(pg),
+        generateDocumentRetrievalReport(pg),
+      ]);
+
+    console.log(JSON.stringify({ patientDiscoveryReport }, null, 2));
+    console.log(JSON.stringify({ documentQueryReport }, null, 2));
+    console.log(JSON.stringify({ documentRetrievalReport }, null, 2));
```

Comment:
Unless we absolutely need to split this in diff log entries, let's not format on JSON.stringify.

I think it would actually make it harder to copy/paste if we format it.